src/miradorDownloadPlugin.js

```diff
@@ -1,17 +1,85 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import ListItemText from '@material-ui/core/ListItemText';
+import MenuItem from '@material-ui/core/MenuItem';
+import Typography from '@material-ui/core/Typography';
+import DownloadIcon from '@material-ui/icons/VerticalAlignBottomSharp';
+import { getCanvasLabel } from 'mirador/dist/es/src/state/selectors/canvases';
+
+const mapStateToProps = (state, { windowId }) => ({
+  title: getCanvasLabel(state, { windowId }),
```

Comment:
We'll most likely need to change this to current canvases later, but just wanted to keep it simple for now.